at_client/lib/src/client/at_client_impl.dart

```diff
@@ -470,7 +470,11 @@ class AtClientImpl implements AtClient {
     var result = <AtKey>[];
     if (getKeysResult.isNotEmpty) {
       getKeysResult.forEach((key) {
-        result.add(AtKey.fromString(key));
+        try {
+          result.add(AtKey.fromString(key));
+        } on InvalidSyntaxException {
```

Comment:
Better to handle a broader exception here

Reply:
I mean an additional catch block for generic exceptions with message 

_logger.severe('Unable to form key $key');

Reply:
Sure Jagan.